### PR TITLE
Bugfix/button permissions

### PR DESCRIPTION
**Pull request by thimo-schneider**
# Submit a pull request

Hier wurde bisher nur der zweite Teil des Tasks umgesetzt, da erst noch festgestellt werden muss ob der erste Teil nicht bereits umgesetzt wurde.

## Please make sure the following is true:

- [x] I gave the PR a meaningful name
- [x] I checked that the correct target branch is selected
- [x] I rebased the branch on the target branch and it can be merged
- [x] I ran the linter and it did pass
- [x] I checked for unused code / dead code / debug code
- [x] I checked that variables/functions have meaningful names
- [x] I checked that the behaviour is as the documentation/task describes and I tested it
- [x] I updated the docs / specifications if possible
- [x] I could explain all that code when someone wakes me up at 3am
- [x] I checked that the code considers failures and not just the happy path
- [x] There are no new dependencies OR I listed them and explained them below
- [x] PR introduces no breaking changes OR I listed them and described them below
- [x] I added/updated tests for new/modified unit-testable functions/helpers
- [x] I ran the tests and they did pass

## Reason for this PR
AC-Task: https://app.activecollab.com/116706/projects/2?modal=Task-46156-2

### CODE REVIEW

File: src/app/helpers/accessManagementHelper.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ const _lookUpPermissions = params => {
     const state = store.getState();
     const tables = state.tables;
     const columns = state.columns[tblId];
-    const rows = (tblId && state.rows[tblId].data) || {};
+    const rows = (tblId && state.rows[tblId]?.data) || {};
```
Comment (hermann-p) [inline src/app/helpers/accessManagementHelper.js:30]: Das geht ohne TypeScript?

Comment (thimo-schneider) [reply src/app/helpers/accessManagementHelper.js:30]: Jep, war auch überrascht: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining#browser_compatibility

Comment (Zwergal) [reply src/app/helpers/accessManagementHelper.js:30]: @hermann-p Ist das hier eigentlich resolved? Wenn ja, dann könnte man das auch testen + mergen, oder gibt es noch Einwände von Dir?

**Comment by Zwergal** [comment]
@hermann-p Hier fehlt noch eine Antwort/Review von Dir!

**Comment by hermann-p** [comment]
Out of date, funktionalität in #356 integriert.

**Pull request closed without merging**